salt/modules/disk.py

```diff
@@ -268,24 +268,33 @@ def percent(args=None):
 
 
 @salt.utils.decorators.path.which('blkid')
-def blkid(device=None):
+def blkid(device=None, token=None):
     '''
     Return block device attributes: UUID, LABEL, etc. This function only works
     on systems where blkid is available.
 
+    device
+        Device name from the system
+
+    token
+        Any valid token used for the search
+
     CLI Example:
 
     .. code-block:: bash
 
         salt '*' disk.blkid
         salt '*' disk.blkid /dev/sda
+        salt '*' disk.blkid UUID=6a38ee5-7235-44e7-8b22-816a403bad5d
```

Comment:
Is this the example intended to illustrate the use of a token? If so, I think we should make this more clear.

Reply:
Ouch. Yes.